types/qrcode/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for qrcode 0.8.2
+// Type definitions for qrcode 1.2.0
```

Comment:
Looks like our lint rules are disabled for this package. Can you enable the lints in our `tslint.json` and fix up the version string to only be a `MAJOR.MINOR` version (e.g. `4.7`), as opposed to including the patch or containing a leading `v` (e.g. `v1.2.3`).

Alternatively, if it is too much, at least just re-enable the header lint rules

Reply:
Just a gently ping - if disabling *all* the rules is too much, feel free to just disable the header parsing rule and make the suggested change.

Reply:
Done - just hadn't had a chance to get around to it.

Couldn't enable `unified-signatures` because of some overloads that take `any` as a substitute for a `node-canvas` canvas instance. I didn't want to go about making an interface for a `NodeCanvasLike` type since I don't know what all is needed there.